### DISCUSSION TITLE
Welcome to deb.sury.org Discussions!

### DISCUSSION THREAD

**Discussion by oerdnj** [oerdnj/deb.sury.org#1696] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ewasser** [comment]
Hi,

I'm Erik and the customers of my employer are willingly using old PHP versions for their online shops.

> Customer: Never change a running system.

So, your work keeps the online (business) world safe.

Thank you for your never-ending effort to care about old PHP versions. Keep up the good work.
